### ISSUE TITLE
"multiple definition of" errors when linking with gcc 10.1

### ISSUE THREAD

**Issue by M-Reimer** [stefantalpalaru/w_scan2#24]
I get a lot of messages like these when compiling/linking w_scan2 with gcc 10.1:

```
/usr/bin/ld: src/descriptors.o:(.bss+0x0): multiple definition of `p_transponder_t'; src/countries.o:(.bss+0x0): first defined here
/usr/bin/ld: src/descriptors.o:(.bss+0x20): multiple definition of `transponder_t'; src/countries.o:(.bss+0x20): first defined here
/usr/bin/ld: src/descriptors.o:(.bss+0xe8): multiple definition of `p_service_t'; src/countries.o:(.bss+0xe8): first defined here
/usr/bin/ld: src/descriptors.o:(.bss+0x100): multiple definition of `service_t'; src/countries.o:(.bss+0x100): first defined here
/usr/bin/ld: src/diseqc.o:(.bss+0x8): multiple definition of `p_transponder_t'; src/countries.o:(.bss+0x0): first defined here
/usr/bin/ld: src/diseqc.o:(.bss+0x20): multiple definition of `transponder_t'; src/countries.o:(.bss+0x20): first defined here
/usr/bin/ld: src/diseqc.o:(.bss+0xe8): multiple definition of `p_service_t'; src/countries.o:(.bss+0xe8): first defined here
/usr/bin/ld: src/diseqc.o:(.bss+0x100): multiple definition of `service_t'; src/countries.o:(.bss+0x100): first defined here
/usr/bin/ld: src/dump-dvbscan.o:(.bss+0x0): multiple definition of `p_transponder_t'; src/countries.o:(.bss+0x0): first defined here
/usr/bin/ld: src/dump-dvbscan.o:(.bss+0x20): multiple definition of `transponder_t'; src/countries.o:(.bss+0x20): first defined here
/usr/bin/ld: src/dump-dvbscan.o:(.bss+0xe8): multiple definition of `p_service_t'; src/countries.o:(.bss+0xe8): first defined here
/usr/bin/ld: src/dump-dvbscan.o:(.bss+0x100): multiple definition of `service_t'; src/countries.o:(.bss+0x100): first defined here
/usr/bin/ld: src/dump-vdr.o:(.bss+0x0): multiple definition of `p_transponder_t'; src/countries.o:(.bss+0x0): first defined here
/usr/bin/ld: src/dump-vdr.o:(.bss+0x20): multiple definition of `transponder_t'; src/countries.o:(.bss+0x20): first defined here

```

**Comment by stefantalpalaru** [comment]
Must be https://wiki.gentoo.org/wiki/Gcc_10_porting_notes/fno_common

**Comment by M-Reimer** [comment]
Most probably

**Comment by stefantalpalaru** [comment]
Fixed in 1.0.9
